funflow/src/Control/Funflow.hs

```diff
@@ -1,3 +1,6 @@
+-- | Central Funflow module.
+--
+--   This module just re-exports various other modules for convenience.
```

Comment:
This would be a good place to hang an example or even a full-fledged mini-tutorial.